.circleci/config.yml

```diff
@@ -78,25 +74,21 @@ jobs:
     steps:
       - run: *install-git
       - checkout
-      - python/install-packages:
-          pkg-manager: pip
       - run:
           name: wake up serverless db
           command: |
             cd test/integration
             dbt debug --target azuresql_sqlcred
       - run:
           name: Test adapter on Azure SQL against dbt-adapter-tests
-          command: tox -e integration-azuresql
+          command: tox -- -v test/integration/azuresql.dbtspec
   connection-azuresql:
```

Comment:
I think we still need to install the dependencies here. for `connection-azuresql` and `connection-sqlserver` I'm cheating by using not using `tox` or `pytest`, instead, we:
- install the dependencies for `dbt-sqlserver` using the `python/install-packages` CirceCI command
- install `gnupg` (note: this should be added to the docker container)
- install the azure-cli,
- move to the dir of the integration test's `dbt_project.yml`
- call `dbt compile` to debug a connection with:
    - a SQL cred
    - an AAD account, and
    - service principal credential